tests/tests-connect.js

```diff
@@ -0,0 +1,65 @@
+import test from 'ava'
+import sleep from 'sleep-promise'
+
+import sbClient, { SpacebroClient } from '../src/spacebro-client'
+
+test('Simple connect', async (t) => {
+  sbClient.connect('spacebro.space', 3333, {
```

Comment:
The other tests all use instanced SpacebroClient object; I left two tests with the legacy version because gabrielstuff wanted the tests to confirm that the legacy API did work (since it's used in all current spacebro projects).